### DISCUSSION TITLE
PnP Site Provisioning Status???

### DISCUSSION THREAD

**Discussion by myatix** [pnp/powershell#3812] [Q&A]
Hi All,

We have implemented a PnP Site Provisioning solution using an Azure Function App that applies a PnP Template stored in a SharePoint document library. It can take some time for certain PnP Templates to apply and I was wondering if anyone new of a smart way to show the user of the site that their site was currently being provisioned?

Perhaps adding a banner to the top of the site or something similar, so they don't start editing any pages until the PnP Site Template is fully deployed... (Similar to the one when using Site Designs in SPO perhaps???

I hope someone can provide some insights into a good solution to this issue?

**Comment by BaronSparky** [comment]
How about inserting a page section/text into the sites homepage before applying the template and then removing once complete?
(https://pnp.github.io/powershell/cmdlets/Add-PnPPageSection.html)
